.github/workflows/lint-and-test.yml

```diff
@@ -22,4 +22,5 @@ jobs:
       - run: npm run build:esm
       - run: npm test
       - run: npm run lint
+      - run: npm run prettier
```

Comment:
kinda nice to have diff actions for lint / prettier and testing, but I don't have a strong opinion on that

Reply:
Actually, I think you're right. I'm going to push that update, thanks for flagging.